pkg/test/framework/components/istio/operator.go

```diff
@@ -251,23 +251,24 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		}
 	}
 
-	// Deploy Istio to remote clusters
+	// Deploy base to remote clusters so create remote secret won't fail (need service account)
```

Comment:
is this what a user will need to do? Deploy base, then remote secret, then the full thing?

Reply:
As of today with JWT auth enabled yes. We need a better way to do things. Is there a better `istioctl` command for setting up just the reader service account? That's the only thing that prevents us from running `create-remote-secret` before installing.